dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporter.java

```diff
@@ -13,57 +13,80 @@
  */
 package com.dynatrace.opentelemetry.metric;
 
+import com.dynatrace.metric.util.Dimension;
+import com.dynatrace.metric.util.DimensionList;
+import com.dynatrace.metric.util.MetricBuilderFactory;
 import com.google.common.annotations.VisibleForTesting;
-import io.opentelemetry.api.metrics.common.Labels;
+import com.google.common.base.Charsets;
+import com.google.common.base.Joiner;
+import com.google.common.base.Strings;
+import com.google.common.io.CharStreams;
 import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.metrics.data.AggregationTemporality;
 import io.opentelemetry.sdk.metrics.data.MetricData;
 import io.opentelemetry.sdk.metrics.export.MetricExporter;
+import java.io.InputStreamReader;
 import java.io.OutputStream;
 import java.net.HttpURLConnection;
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.nio.charset.StandardCharsets;
-import java.util.AbstractMap;
 import java.util.ArrayList;
 import java.util.Collection;
+import java.util.List;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import javax.annotation.Nonnull;
 
 /** Export metric to Dynatrace. */
 public final class DynatraceMetricExporter implements MetricExporter {
   private final URL url;
   private final String apiToken;
+  private final Serializer serializer;
 
   private static final Logger logger = Logger.getLogger(DynatraceMetricExporter.class.getName());
+  private static final List<Dimension> staticDimensions =
+      new ArrayList<Dimension>() {
+        {
+          add(Dimension.create("dt.metrics.source", "opentelemetry"));
+        }
+      };
+
+  private static final Pattern EXTRACT_LINES_OK = Pattern.compile("\"linesOk\":\\s?(\\d+)");
```

Comment:
Yes, it would, but it would also require importing external libraries (in this case we could use Jackson for example). I copied this from the micrometer exporter, where we are trying to use as little external libraries as possible, and where we only had a reference to Jackson for testing, but not the implementation. It would definitely be the more elegant and robust solution, but i am not sure if we want to add a dependency just for logging.

Reply:
It's fine. I always forget other languages don't have built in json support.